dev/DataSource/Mapping/TopLevelDomainMapping.php

```diff
@@ -0,0 +1,91 @@
+<?php
+declare(strict_types=1);
+
+namespace PrinsFrank\Standards\Dev\DataSource\Mapping;
+
+use Facebook\WebDriver\WebDriverBy;
+use PrinsFrank\Standards\Dev\DataSource\Sorting\KeyWithDeprecatedTagsSeparateSorting;
+use PrinsFrank\Standards\Dev\DataSource\Sorting\SortingInterface;
+use PrinsFrank\Standards\Dev\DataTarget\EnumCase;
+use PrinsFrank\Standards\Dev\DataTarget\EnumFile;
+use PrinsFrank\Standards\InvalidArgumentException;
+use PrinsFrank\Standards\TopLevelDomain\CountryCodeTLD;
+use PrinsFrank\Standards\TopLevelDomain\GenericRestrictedTLD;
+use PrinsFrank\Standards\TopLevelDomain\GenericTLD;
+use PrinsFrank\Standards\TopLevelDomain\InfrastructureTLD;
+use PrinsFrank\Standards\TopLevelDomain\SponsoredTLD;
+use PrinsFrank\Standards\TopLevelDomain\TestTLD;
+use stdClass;
+use Symfony\Component\Panther\Client;
+use Symfony\Component\Panther\DomCrawler\Crawler;
+
+/**
+ * @template TDataSet of object{tld: string, type: string, manager: string}&stdClass
+ * @implements Mapping<TDataSet>
+ */
+class TopLevelDomainMapping implements Mapping
+{
+    public static function url(): string
+    {
+        return 'https://www.iana.org/domains/root/db';
+    }
+
+    /** @return list<TDataSet> */
+    public static function toDataSet(Client $client, Crawler $crawler): array
+    {
+        $items = $crawler->filterXPath('//table[@id="tld-table"]/tbody/tr')->getIterator();
+
+        $dataSet = [];
+        foreach ($items as $item) {
+            $columns = $item->findElements(WebDriverBy::xpath('./td'));
+            if (count($columns) !== 3) {
+                continue;
+            }
+
+            $record          = (object) [];
```

Comment:
Without hacking it is really
```suggestion
            $record          = new stdClass();
```